src/storage/src/hummock/event_handler/uploader/mod.rs

```diff
@@ -253,7 +261,9 @@ impl UploadingTask {
         }
         let join_handle = (context.spawn_upload_task)(payload, task_info.clone());
         context.stats.uploader_uploading_task_count.inc();
+        static NEXT_TASK_ID: LazyLock<AtomicUsize> = LazyLock::new(|| AtomicUsize::new(0));
```

Comment:
ditto: only `task_manager` can new a `UploadingTask`. Would it be clearer if `task_manager` takes care of the id generation internally without using `LazyLock<AtomicUszie>`?